### DISCUSSION TITLE
General advice on training a custom dataset w/ GlowTTS

### DISCUSSION THREAD

**Discussion by T145** [coqui-ai/TTS#3416] [General Q&A]
So I have a custom dataset w/ audio clips that range from a few seconds to a couple minutes. Its base sample rate is 48000, which I resample to 22050 using the `resample` script:
```python
from TTS.bin.resample import resample_files
resample_files(sounds, 22050, file_ext='wav', n_jobs=10)
```
This is the only data preparation I perform as I assume audio with a lower quality is easier for the trainer to work with. To train a model I'm using the script given in the [official tutorial](https://colab.research.google.com/github/coqui-ai/TTS/blob/dev/notebooks/Tutorial_2_train_your_first_TTS_model.ipynb), except I boosted the epoch count from 100 to 300.

Performing text-to-speech using the best model and a source WAV is fairly robotic, so what should be done to make it closer to the original quality? Are there any recommended parameters to add or remove in the config to help, or more that should be done to the audio itself?

EDIT: The model is `en` and not multi-lingual.

**Comment by ianujkr** [comment]
Hello,
 Did u get your model trained on the mentioned model and what was the speed with which model was getting trained on?

**Reply by T145** [reply]
As mentioned I trained it with the parameters given in the official tutorial. Everything was the same but the epoch count.
